index.js

```diff
@@ -278,6 +267,14 @@ function transformObservationSchema1 (obs) {
   return newObs
 }
 
+function toObs (elm) {
+  // TODO: sometimes the version is not here why?
```

Comment:
createReadStream from the kappa-core writer doesnt have a version

Reply:
ah ha, yes! `writer` is a raw hypercore object, so you're getting the raw hypercore data /wo any nice kappa-osm massaging. This object should probably be considered a "private" API..

I think it makes sense to expose an API on kappa-osm to get a stream of objects by type. Let's hack this in for now (massage the data format) and do a real API later

Reply:
aaah. Also, this will return *all* entries in the hypercore, including non-heads. I'll think about this...

Reply:
Yeah, I'll write a new index for kappa-osm to return a stream of observation heads!

Reply:
added a `byType(type)` api to kappa-osm for this

Reply:
added a `byType(type)` api to kappa-osm for this